### PR TITLE
Fix impots and paths

### PR DESCRIPTION
**Pull request by kevinvigil**

### CODE REVIEW

File: src/main/java/com/system/billingsystem/dto/CompanyDto.java [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
-import com.system.billingSystem.entities.Invoice;
+import com.system.billingsystem.entities.Invoice;
 
 import java.util.List;
 import java.util.UUID;
 
 public record CompanyDto(
-        UUID CompanyDto_id,
+        UUID companyDtoId,
```
Comment (matiasvigil) [inline src/main/java/com/system/billingsystem/dto/CompanyDto.java]:
```suggestion
        UUID companyId,
```

File: src/main/java/com/system/billingsystem/dto/CompanyDto.java [hunk 2]
```diff
@@ -30,9 +30,8 @@ public  CompanyDto (UUID CompanyDto_id) {
         this (CompanyDto_id, null, null, null, null, null, null, null);
     }
 
-    @Override
-    public UUID CompanyDto_id() {
-        return CompanyDto_id;
+    public UUID companyDtoId() {
```
Comment (matiasvigil) [inline src/main/java/com/system/billingsystem/dto/CompanyDto.java]:
```suggestion
    public UUID companyId() {
```

File: src/main/java/com/system/billingsystem/dto/InvoiceDto.java [hunk 3]
```diff
@@ -7,7 +7,7 @@
 import java.util.UUID;
 
 public record InvoiceDto (
-        UUID invoiceDto_id,
+        UUID invoiceDtoId,
```
Comment (matiasvigil) [inline src/main/java/com/system/billingsystem/dto/InvoiceDto.java]:
```suggestion
        UUID invoiceId,
```

File: src/main/java/com/system/billingsystem/dto/InvoiceProductDto.java [hunk 4]
```diff
@@ -1,17 +1,17 @@
-package com.system.billingSystem.dto;
+package com.system.billingsystem.dto;
 
 import java.util.Objects;
 import java.util.UUID;
 
 public record InvoiceProductDto (
-        UUID invoiceProductDto_id,
+        UUID invoiceProductDtoId,
```
Comment (matiasvigil) [inline src/main/java/com/system/billingsystem/dto/InvoiceProductDto.java]:
```suggestion
        UUID invoiceProductId,
```

File: src/main/java/com/system/billingsystem/dto/InvoiceProductDto.java [hunk 5]
```diff
@@ -1,17 +1,17 @@
...
         String name,
         Integer amount,
         UUID idProduct,
         UUID idInvoice ) {
```
Comment (matiasvigil) [inline src/main/java/com/system/billingsystem/dto/InvoiceProductDto.java]:
```suggestion
        UUID productId,
        UUID invoiceId ) {
```

File: src/main/java/com/system/billingsystem/dto/ProductDto.java [hunk 6]
```diff
@@ -1,24 +1,23 @@
-package com.system.billingSystem.dto;
+package com.system.billingsystem.dto;
 
 import java.math.BigDecimal;
 import java.util.UUID;
 
 public record ProductDto (
-        UUID productDto_id,
+        UUID productDtoId,
```
Comment (matiasvigil) [inline src/main/java/com/system/billingsystem/dto/ProductDto.java]:
```suggestion
        UUID productId,
```

File: src/main/java/com/system/billingsystem/entities/InvoiceProduct.java [hunk 7]
```diff
@@ -1,17 +1,16 @@
...
 
-import java.math.BigDecimal;
 import java.util.UUID;
 
 @Data
 @Builder
 @AllArgsConstructor
 public class InvoiceProduct {
-    private UUID invoiceproduct_id;
+    private UUID invoiceproductId;
```
Comment (matiasvigil) [inline src/main/java/com/system/billingsystem/entities/InvoiceProduct.java]:
```suggestion
    private UUID invoiceProductId;
```

File: src/main/java/com/system/billingsystem/entities/InvoiceProduct.java [hunk 8]
```diff
@@ -24,9 +23,9 @@ public InvoiceProduct() {}
     @Override
     public String toString(){
         return ("InvoiceProduct { " +
-                ", invoiceProduct_id: " + this.invoiceproduct_id +
-                ", invoice_id: " + ((this.invoice == null)? null:this.invoice.getInvoice_id()) +
-                ", product_id: " + ((this.product == null)? null: this.product.getProduct_id()) +
+                ", invoiceProduct_id: " + this.invoiceproductId +
+                ", invoice_id: " + ((this.invoice == null)? null:this.invoice.getInvoiceId()) +
+                ", product_id: " + ((this.product == null)? null: this.product.getProductId()) +
                 ", amount: " + this.amount +
```
Comment (matiasvigil) [inline src/main/java/com/system/billingsystem/entities/InvoiceProduct.java]: amount should be BigDecimal

File: src/main/java/com/system/billingsystem/entities/InvoiceVoucher.java [hunk 9]
```diff
@@ -1,5 +1,8 @@
-package com.system.billingSystem.entities;
+package com.system.billingsystem.entities;
 
+import org.graalvm.nativeimage.c.constant.CEnum;
```
Comment (matiasvigil) [inline src/main/java/com/system/billingsystem/entities/InvoiceVoucher.java]: What is this for?

Comment (kevinvigil) [reply src/main/java/com/system/billingsystem/entities/InvoiceVoucher.java:3]: With out this Spring Boot does not read this enum and throw the error "simbol not find". The same problem apare on BaseRepository with out "@Repository".

File: src/main/java/com/system/billingsystem/entities/InvoiceProduct.java (hunk 8 above)
Comment (kevinvigil) [reply src/main/java/com/system/billingsystem/entities/InvoiceProduct.java]: InviceProduct.amount referees a quantity of some product. 
No have sense on have 1,5 of product. 
Should I change the name of the variable?

Comment (matiasvigil) [reply src/main/java/com/system/billingsystem/entities/InvoiceProduct.java]: Yes, it is not `amount` but `count` then.

**Pull request closed without merging**